src/com/projectkorra/projectkorra/firebending/BlazeArc.java

```diff
@@ -40,6 +34,11 @@ public BlazeArc(final Player player, final Location location, final Vector direc
 		this.range = this.getDayFactor(range);
 		this.speed = getConfig().getLong("Abilities.Fire.Blaze.Speed");
 		this.interval = (long) (1000. / this.speed);
+
+		if(bPlayer.canUseSubElement(SubElement.BLUE_FIRE)) {
+			this.range += BlueFireAbility.getRangeFactor() * range;
```

Comment:
I think `+=` should just be `=`. This will at least double the value every time, not just multiply it by the modifer.

Reply:
Aha, yes, my bad I was going to make it so that it only adds the difference between the base and the modifier, thanks for pointing it out